docs/settings/task-manager-settings.asciidoc

```diff
@@ -48,6 +49,9 @@ Enables event loop delay monitoring, which will log a warning when a task causes
 `xpack.task_manager.event_loop_delay.warn_threshold`::
 Sets the amount of event loop delay during a task execution which will cause a warning to be logged. Defaults to 5000 milliseconds (5 seconds).
 
+`xpack.task_manager.capacity`::
+Sets the number of normal cost tasks that can be run at one time. Can be minimum 5 and maximum 50. Defaults to 10.
```

Comment:
What about something like `controls the number of tasks that can be run at one time`? cc @lcawl 

Wording can be confusing because the capacity means something different depending on what the claim strategy is. It is either the number of tasks that can be run at one time or the total cost of tasks that can be run at one time.

Reply:
I update the text in this commit [baf8040](https://github.com/elastic/kibana/pull/192124/commits/baf80406520239ece9fa0853479e4ed152a195d7)